protected/views/project/summary.php

```diff
@@ -338,7 +386,10 @@
     }
 
     if (!empty($project->pages) && (true || \Yii::$app->user->can(Permission::PERMISSION_READ, $project))) {
-       echo Html::a('Details', ['project/view', 'id' => $project->id], ['target' => '_top']);
+        echo Html::beginTag('div', ['class' => 'actions']);
```

Comment:
The buttons are in the grid, there should be no need for extra tags just to create boxes?